legal-api/migrations/versions/b14f77400786_xml_payloads.py

```diff
@@ -0,0 +1,43 @@
+"""xml_payloads
+
+Revision ID: b14f77400786
+Revises: feb206b2ce65
+Create Date: 2024-07-17 23:46:20.873233
+
+"""
+from alembic import op
+import sqlalchemy as sa
+
+
+# revision identifiers, used by Alembic.
+revision = 'b14f77400786'
+down_revision = 'feb206b2ce65'
+branch_labels = None
+depends_on = None
+
+
+def upgrade():
+    op.create_table('xml_payloads',
+    sa.Column('id', sa.Integer(), nullable=False),
+    sa.Column('payload', sa.TEXT, nullable=False),    
```

Comment:
I was envisioning us using the postgres xml column type as it may have better performance and features specific to xml.  

Can you check if the following works?  

`sa.Column('payload', sa.XML(), nullable=False),`

Reply:
But, `sa` doesn't support the XML
<img width="638" alt="Screenshot 2024-07-19 at 1 30 44 PM" src="https://github.com/user-attachments/assets/5f1cd321-b9f0-4830-a580-eecd17b811e2">

Reply:
give this a try - `sa.Column('payload', postgresql.XML(), nullable=False),`

I think you'll need to do any import `from sqlalchemy.dialects import postgresql`

Reply:
I tried, but got this error
`AttributeError: module 'sqlalchemy.dialects.postgresql' has no attribute 'XML'`

Reply:
I'm pretty sure XML should be supported for postgres.

I may not have given an example that works but can you try figuring how to get XML column type working?

Reply:
I tried to figure it out. But, didn't find the solution. ☹️

Reply:
Postgres supports XML while SQLAlchemy doesn't support it dirrectly. Maybe you can try to create a custom SQLAlchemy type to handle XML. 🤔

Reply:
yes, please try Hongjing's suggestion, @kzdev420

Reply:
I created the XML data type.

<img width="657" alt="Screenshot 2024-07-24 at 8 27 15 AM" src="https://github.com/user-attachments/assets/307250a6-3692-479a-b557-6478a5ee5245">